README.md

```diff
@@ -172,11 +179,15 @@ pkill -f selenium-standalone
 
 ## Available browsers
 
+---
+
 By default, google chrome, firefox and phantomjs are available
 when installed on the host system.
 
 ## Tips
 
+---
```

Comment:
Oh! That was semi-intentionally me - The separator made the sections pop-out a bit more, though that might just be me.

Totally okay with reverting that change if desired!